xml/System.Net.Security/ServerCertificateSelectionCallback.xml

```diff
@@ -29,10 +29,11 @@
     <ReturnType>System.Security.Cryptography.X509Certificates.X509Certificate</ReturnType>
   </ReturnValue>
   <Docs>
-    <param name="sender">To be added.</param>
-    <param name="hostName">To be added.</param>
-    <summary>To be added.</summary>
-    <returns>To be added.</returns>
-    <remarks>To be added.</remarks>
+    <param name="sender">An object that contains state information for this validation.</param>
+    <param name="hostName">The host name requested by the client.</param>   
+    <summary>Selects the server Secure Sockets Layer (SSL) certificate.</summary>
+    <returns>An <see cref="T:System.Security.Cryptography.X509Certificates.X509Certificate" /> used for establishing an SSL connection.</returns>
+    <remarks>This delegate is used by the the <xref:System.Net.Security.SslStream> class. The <xref:System.Net.Security.SslStream> class is used to help secure information exchanged between a client and server. The server uses this delegate to select a certificate to prove its identity.      
```

Comment:
I'm not sure if we need to describe purpose of SslStream here. I would add note that the certificate must include private key. Without it , the AuthenticateAsServer will throw. It will also throw if callback returns null.

Reply:
Correct format for the remarks section

```suggestion
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  

This delegate is used by the <xref:System.Net.Security.SslStream> class. The <xref:System.Net.Security.SslStream> class is used to help secure information exchanged between a client and server. The server uses this delegate to select a certificate to prove its identity.      
  
 ]]></format>
```